quickstarts/aws/amazon-workspaces/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: amazon-workspaces
+description: |-
+  ## What is Amazon WorkSpaces?
+  Amazon WorkSpaces is a fully managed desktop virtualization service for Windows and Linux that enables you to access resources from any supported device.
+  ### Get started!
+  Start monitoring Amazon WorkSpaces by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-workspaces-monitoring-integration/) to learn more about New Relic monitoring for Amazon WorkSpaces.
+summary: |-
+  Monitor Amazon WorkSpaces by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Amazon WorkSpaces
+documentation:
+  - name: Amazon WorkSpaces installation docs
+    description: |
+      Monitor Amazon WorkSpaces by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-WorkSpaces-monitoring-integration/
```

Comment:
Observed this documentation link is not live.

Reply:
@pkosanam I have updated the path. its working now